addons/website/models/ir_http.py

```diff
@@ -133,7 +133,7 @@ def _auth_method_public(cls):
 
     @classmethod
     def _register_website_track(cls, response):
-        if getattr(response, 'status_code', 0) != 200:
+        if getattr(response, 'status_code', 0) != 200 or request.httprequest.headers.get('X-Disable-Tracking') == '1':
```

Comment:
Would be nice to have this change isolated in its own commit with an explanation that we introduce it to make it possible to skip the tracking of certain pages / use cases (you can give the service worker example).

Reply:
Good idea, it is done.